controllers/iamauthpolicy_controller.go

```diff
@@ -43,6 +47,12 @@ func (c *IAMAuthPolicyController) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 	c.log.Infow("reconcile", "req", req, "targetRef", k8sPolicy.Spec.TargetRef)
 
+	err = c.handleConflicts(ctx, k8sPolicy)
+	if err != nil {
+		c.log.Error(err)
+		return ctrl.Result{RequeueAfter: 30 * time.Second}, nil
```

Comment:
for the Conflicted policy, should it just be "reconciled" with PolicyReasonConflicted status, instead of keep looping?

Reply:
with loop it should converge by itself when conflict is resolved